index.js

```diff
@@ -3,11 +3,14 @@ const ManyKeysMap = require('many-keys-map');
 const pDefer = require('p-defer');
 
 const cache = new ManyKeysMap();
-const isDomReady = () => document.readyState === 'interactive' || document.readyState === 'complete';
+const isDomReady = target =>
+	(target.ownerDocument || target).readyState === 'interactive' ||
+	(target.ownerDocument || target).readyState === 'complete';
 
 const elementReady = (selector, {
 	target = document,
 	stopOnDomReady = true,
+	expectEntireElement = true,
```

Comment:
If I default this to `false` it won't be a breaking change, but I think `true` is the correct expectation, as discussed in #29 

Additional possible breaking changes before a release:

- async functions (not sure if needed) or generally increasing the minimum supported version
- TypeScript conversion

And also

- I could look into dropping `p-deferred` and possibly reworking the awkward `stop(); return;` repetition

Reply:
> If I default this to false it won't be a breaking change, but I think true is the correct expectation, as discussed in #29

I agree it's worth doing a breaking release for this.

> TypeScript conversion

I don't think it's worth the overhead to convert this tiny package to TS. I'm generally conservative about TS. It introduces a lot of problems too.